subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/ContinuousBuildFileWatchingIntegrationTest.groovy

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.watch
+
+import org.gradle.integtests.fixtures.AbstractContinuousIntegrationTest
+import org.gradle.integtests.fixtures.FileSystemWatchingFixture
+import org.gradle.integtests.fixtures.FileSystemWatchingHelper
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import spock.lang.IgnoreIf
+
+@IgnoreIf({ GradleContextualExecuter.noDaemon || GradleContextualExecuter.watchFs }) // See AbstractFileSystemWatchingIntegrationTest
```

Comment:
```suggestion
@IgnoreIf({
	GradleContextualExecuter.noDaemon    // There is no shared state without a daemon
	|| GradleContextualExecuter.watchFs  // The test manually enables file system watching
})
```

Reply:
I moved the `||` to the end of the first line, if I don't the Groovy compiler thinks the statement already ended in the first line and then fails on the second.